renovate.json

```diff
@@ -1,21 +1,8 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "platform": "github",
-  "labels": ["renovate"],
   "extends": [
-    "config:base",
-    ":rebaseStalePrs"
+    "github>gsoft-inc/renovate-config"
   ],
```

Comment:
- I think we should migrate the config `excludePackagePatterns` to `excludePackageNames`
- Does `^dotnet-sdk$` still make sense

Reply:
I tried matching the most common rules. What is the benefits of excludePackageNames (we would need to update multiple repo if we want to be consistant)

Reply:
> What is the benefits of excludePackageNames

I think that the new way of doing it, but we can do it later. Anyway, Renovate will open PRs to fix it later if needed. I've already merged some of them.